market/uipath/demo/Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+pipeline {
+  agent {
+    docker {
+      image 'axonivy/build-container:web-1.0'

```

Comment:
I don't like this container. But this is another story.

Reply:
I only need @IvyProcessTEst support... is there a better one? ... just copied blindly from 'project-build-examples' :+1: